app/components/help/form_component.html.erb

```diff
@@ -0,0 +1,79 @@
+<div class="modal fade" id="contactUsModal" tabindex="-1" aria-labelledby="content-type-prompt" aria-hidden="true">
```

Comment:
I don't see a corresponding element with an ID of `content-type-prompt` to make sure this is labelled. I wonder if the value of this attribute should be `help-prompt`, in which case you might want to modify line 5 below to:

```html
<h3 class="modal-title" id="help-prompt">Help</h3>
```

Reply:
Good catch; fixed.